modules/nw-running-dpdk-rootless-tap.adoc

```diff
@@ -0,0 +1,260 @@
+<5> Defines the group ID (GID) that the pod's containers use to access shared volumes' file systems. This needs to be set to 801 to access the VFIO device.
++
+[NOTE]
+====
+Alternatively add the group 801 to the non-root user in the image. The user 1001(`dpdk-user`) in `registry.redhat.io/openshift4/dpdk-base-rhel8` has this group added.
+====
++
+<6> Controls which user ID the container is run with.
+<7> Controls which primary group ID the containers is run with.
+<8> This setting indicates that the container(s) within the pod should not be granted privileged access to the host system.
+<9>  This setting allows a container to escalate its privileges beyond the initial non-root privileges it may have been assigned.
+<10> This setting ensures that the container runs with a non-root user. This helps enforce the principle of least privilege, limiting the potential impact of a container compromise and reducing the attack surface.
+<11> Mount a hugepage volume to the DPDK pod under `/dev/hugepages`. The hugepage volume is backed by the emptyDir volume type with the medium being `Hugepages`.
+<12> Optional: Specify the number of DPDK devices allocated to DPDK pod. This resource request and limit, if not explicitly specified, will be automatically added by the SR-IOV network resource injector. The SR-IOV network resource injector is an admission controller component managed by the SR-IOV Operator. It is enabled by default and can be disabled by setting `enableInjector` option to `false` in the default `SriovOperatorConfig` CR.
+<13> Specify the number of CPUs. The DPDK pod usually requires exclusive CPUs to be allocated from the kubelet. This is achieved by setting CPU Manager policy to `static` and creating a pod with `Guaranteed` QoS.
+<14> Specify hugepage size `hugepages-1Gi` or `hugepages-2Mi` and the quantity of hugepages that will be allocated to the DPDK pod. Configure `2Mi` and `1Gi` hugepages separately. Configuring `1Gi` hugepage requires adding kernel arguments to Nodes. For example, adding kernel arguments `default_hugepagesz=1GB`, `hugepagesz=1G` and `hugepages=16` will result in `16*1Gi` hugepages be allocated during system boot.
++
+[NOTE]
+====
+Modify the configuration of your DPDK application to run with user ID `1001` and group ID `1001`. This can be achieved through different approaches depending on the specific requirements and configuration options of your DPDK application.
+====
++
+--
++
+. Create the DPDK pod by running the following command:
++
+[source,terminal]
+----
+$ oc create -f intel-dpdk-pod-rootless.yaml
+----
+
+. Verify that the pod is created by running the following command:
++
+[source,terminal]
+----
+$ oc get pod
+----
++
+.Example output
+[source,terminal]
+----
+NAME       READY   STATUS    RESTARTS   AGE
+test-pod   1/1     Running   0          4m36s
+----
+
+. Log in to the pod by running the following command:
++
+[source,terminal]
+----
+$ oc rsh test-pod
```

Comment:
Looks like useless cmd. Would you like to run specific cmd on rootless dpdk pod ?